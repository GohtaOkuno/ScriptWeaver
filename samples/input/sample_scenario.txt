# 森の中の古い館

## プロローグ

暗い森の奥に、長い間忘れ去られた古い館がひっそりと佇んでいる。
冒険者たちは偶然この館を発見し、中に足を踏み入れることになる。

「何だ、この古い建物は...」
冒険者の一人が呟く。

館の扉は既に朽ち果て、簡単に押し開くことができた。

## 第1章：館の内部

館の中は埃っぽく、蜘蛛の巣が至る所に張られている。
床は古い木材で作られており、歩くたびにギシギシと音を立てる。

正面には大きな階段があり、二階へと続いている。
左手には食堂らしき部屋が見える。
右手には書斎のような部屋の扉が開いている。

### 食堂の調査

食堂には古いテーブルと椅子が置かれている。
テーブルの上には、まだ食べかけの料理が残されている。

「まるで、住人が突然いなくなったようだ」

### 書斎の調査

書斎には数多くの本が並んでいる。
机の上には日記が開かれたまま放置されている。

日記の最後のページには、こう書かれている：

「もう時間がない。『それ』がやって来る。この館から逃げなければ...」

## 第2章：地下室の秘密

館の奥で隠し扉を発見した冒険者たち。
扉の向こうには地下への階段が続いている。

地下室は石造りで、中央に古い祭壇がある。
祭壇の上には不気味な彫像が置かれている。

「この彫像...何かがおかしい」

突然、彫像の目が赤く光り始める。

## エピローグ

冒険者たちは館の秘密を解き明かし、無事に脱出することができた。
しかし、森の奥には他にも謎めいた場所があるに違いない。

次の冒険が彼らを待っている。