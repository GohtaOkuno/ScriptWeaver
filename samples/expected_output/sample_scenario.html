<!DOCTYPE html>
<html lang="ja">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>森の中の古い館</title>
    <style>
        body {
            font-family: 'Noto Sans JP', 'Hiragino Sans', 'Yu Gothic', sans-serif;
            line-height: 1.8;
            max-width: 800px;
            margin: 0 auto;
            padding: 20px;
            background-color: #fafafa;
            color: #333;
        }
        
        h1 {
            color: #2c3e50;
            border-bottom: 3px solid #3498db;
            padding-bottom: 10px;
            font-size: 2.2em;
        }
        
        h2 {
            color: #34495e;
            border-left: 5px solid #3498db;
            padding-left: 15px;
            margin-top: 2em;
            font-size: 1.6em;
        }
        
        h3 {
            color: #5a6c7d;
            font-size: 1.3em;
            margin-top: 1.5em;
        }
        
        p {
            margin-bottom: 1.2em;
            text-indent: 1em;
        }
        
        /* 会話文のスタイル */
        p:has-text("「") {
            font-style: italic;
            color: #7f8c8d;
            background-color: #ecf0f1;
            padding: 10px;
            border-left: 3px solid #bdc3c7;
            margin: 15px 0;
            text-indent: 0;
        }
        
        /* 代替スタイル（CSS3 :has()が使えない場合） */
        .dialogue {
            font-style: italic;
            color: #7f8c8d;
            background-color: #ecf0f1;
            padding: 10px;
            border-left: 3px solid #bdc3c7;
            margin: 15px 0;
            text-indent: 0;
        }
    </style>
</head>
<body>
    <h1>森の中の古い館</h1>
    
    <h2>プロローグ</h2>
    <p>暗い森の奥に、長い間忘れ去られた古い館がひっそりと佇んでいる。</p>
    <p>冒険者たちは偶然この館を発見し、中に足を踏み入れることになる。</p>
    
    <p class="dialogue">「何だ、この古い建物は...」</p>
    <p>冒険者の一人が呟く。</p>
    
    <p>館の扉は既に朽ち果て、簡単に押し開くことができた。</p>
    
    <h2>第1章：館の内部</h2>
    <p>館の中は埃っぽく、蜘蛛の巣が至る所に張られている。</p>
    <p>床は古い木材で作られており、歩くたびにギシギシと音を立てる。</p>
    
    <p>正面には大きな階段があり、二階へと続いている。</p>
    <p>左手には食堂らしき部屋が見える。</p>
    <p>右手には書斎のような部屋の扉が開いている。</p>
    
    <h3>食堂の調査</h3>
    <p>食堂には古いテーブルと椅子が置かれている。</p>
    <p>テーブルの上には、まだ食べかけの料理が残されている。</p>
    
    <p class="dialogue">「まるで、住人が突然いなくなったようだ」</p>
    
    <h3>書斎の調査</h3>
    <p>書斎には数多くの本が並んでいる。</p>
    <p>机の上には日記が開かれたまま放置されている。</p>
    
    <p>日記の最後のページには、こう書かれている：</p>
    
    <p class="dialogue">「もう時間がない。『それ』がやって来る。この館から逃げなければ...」</p>
    
    <h2>第2章：地下室の秘密</h2>
    <p>館の奥で隠し扉を発見した冒険者たち。</p>
    <p>扉の向こうには地下への階段が続いている。</p>
    
    <p>地下室は石造りで、中央に古い祭壇がある。</p>
    <p>祭壇の上には不気味な彫像が置かれている。</p>
    
    <p class="dialogue">「この彫像...何かがおかしい」</p>
    
    <p>突然、彫像の目が赤く光り始める。</p>
    
    <h2>エピローグ</h2>
    <p>冒険者たちは館の秘密を解き明かし、無事に脱出することができた。</p>
    <p>しかし、森の奥には他にも謎めいた場所があるに違いない。</p>
    
    <p>次の冒険が彼らを待っている。</p>
</body>
</html>